app/src/main/java/com/kickstarter/ui/adapters/DiscoveryAdapter.java

```diff
@@ -27,8 +28,9 @@
 public final class DiscoveryAdapter extends KSAdapter {
   private static final int SECTION_ONBOARDING_VIEW = 0;
   private static final int SECTION_EDITORIAL_VIEW = 1;
-  private static final int SECTION_ACTIVITY_SAMPLE_VIEW = 2;
-  private static final int SECTION_PROJECT_CARD_VIEW = 3;
+  private static final int SECTION_LIGHTS_ON_VIEW = 2;
```

Comment:
I was wondering, why can't we use `SECTION_EDITORIAL_VIEW` here?

Reply:
I use it for loading the new view 
<img width="499" alt="Screen Shot 2020-05-06 at 9 01 45 AM" src="https://user-images.githubusercontent.com/4083656/81200119-5d286f80-8f78-11ea-9f0b-5069b9fcd81a.png">